### PR TITLE
Odyssey tab, tag, color, monospace font and ui-indicator updates

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
- [x] Tabs: now uses 16px base font instead of 14px. This now matches the design.
- [x] Tabs: unused CSS custom properties that were used for moving tab indicator implementation have been removed.
- [x] Tabs: Border now lives on the tablist, rather than the tabpanel.
- [x] Tabs: tabpanel padding updated to match the design.
- [x] Tag: tag--list now `inline-block` by default
- [x] Tag: Background color updated to gray 100, per design
- [x] Toast/Modal: Close UI indicator updated to use variant from odyssey-icon
- [x] Now uses Incosolata as the default monospace font for Odyssey, this was recently decided by the brand team! 
- [x] Adds turquoise 400

### CODE REVIEW

File: packages/odyssey/src/scss/abstracts/_functions.scss [hunk 1]
```diff
@@ -72,13 +72,14 @@
...
+  'anchor': '<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M14.264 9.733a4.8 4.8 0 00-6.787 0l-4.36 4.363a4.8 4.8 0 003.393 8.19 4.768 4.768 0 003.39-1.399l3.6-3.6a.343.343 0 00-.243-.585h-.137a5.769 5.769 0 01-2.194-.426.343.343 0 00-.374.076l-2.588 2.591a2.058 2.058 0 11-2.91-2.91L9.43 11.66a2.057 2.057 0 012.907 0c.541.51 1.385.51 1.926 0 .233-.233.374-.542.398-.87a1.371 1.371 0 00-.398-1.056z" fill="%%COLOR%%"/><path d="M20.877 3.12a4.8 4.8 0 00-6.787 0l-3.596 3.592a.343.343 0 00.25.587h.127a5.759 5.759 0 012.19.428.343.343 0 00.374-.075l2.581-2.578a2.058 2.058 0 112.91 2.91l-3.215 3.212-.027.031-1.125 1.118a2.057 2.057 0 01-2.907 0 1.406 1.406 0 00-1.926 0 1.371 1.371 0 00-.398.877c-.025.393.12.777.398 1.056a4.76 4.76 0 001.37.96c.073.034.145.062.217.092.072.031.147.055.22.083.071.027.147.051.219.072l.202.054c.137.035.274.062.415.086.169.025.34.041.51.048h.26l.207-.024c.075-.003.154-.02.243-.02h.117l.236-.035.11-.02.199-.041h.037a4.799 4.799 0 002.225-1.262l4.364-4.364a4.8 4.8 0 000-6.787z" fill="%%COLOR%%"/></svg>',
```
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/abstracts/_functions.scss:75]: Does this still need to go through SVGO? (the `width/height` should have been stripped if so)

File: packages/odyssey/src/scss/abstracts/_tokens.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 // Typography
 $body-font-family: 'Whyte', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
-$mono-font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+$mono-font-family: 'Inconsolata', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/abstracts/_tokens.scss:17]: Good with this even if we can't ship it at present - progressive enhancement where available.

File: packages/odyssey/src/scss/components/_tag.scss [hunk 3]
```diff
@@ -17,11 +17,13 @@
   margin: 0 $spacing-xs $spacing-xs 0;
   padding: 0 $spacing-xs-em;
   border-radius: $base-border-radius;
-  background: cv('gray', '200');
+  background: cv('gray', '100');
   color: $text-body;
 }
 
 .ods-tag--list {
+  display: inline-block;
```
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/components/_tag.scss:25]: I don't think we should change this to account for a horizontal label unless we're going to ship the label itself or include additional guidance around Parents. I'd prefer if we added a Horiz variant / horiz support separately.

File: packages/odyssey/src/scss/abstracts/_functions.scss (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss:75]: Nope, It should have been updated - I ran it through the UI. Will give it another look.

File: packages/odyssey/src/scss/components/_tag.scss (hunk 3 above)
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/components/_tag.scss:25]: My understanding is that this is as @joejezowski-okta had intended for it to be by default, per the design. I will revert this for now and will schedule a follow-up chat/log an issue.

File: packages/odyssey/src/scss/abstracts/_functions.scss (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss]: Fixed.

**Review by edburyenegren-okta — APPROVED** [review]
🚀 

thanks for being flexible while we nail down a lot of moving parts @arnoldsandoval-okta

File: packages/odyssey/src/scss/abstracts/_functions.scss [hunk 4]
```diff
@@ -72,13 +72,14 @@
...
   'check':'<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><path fill="%%COLOR%%" d="M0 5.6L1.6 4l3.2 3.3 5.6-5.7L12 3.1l-7.2 7.4L0 5.6z"/></svg>',
-  'close':'<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16"><path fill="%%COLOR%%" fill-rule="evenodd" d="M13.704 2.295a1.008 1.008 0 0 0-1.426 0L8 6.574 3.722 2.295a1.008 1.008 0 1 0-1.426 1.427L6.574 8l-4.279 4.278a1.008 1.008 0 1 0 1.426 1.426L8 9.427l4.278 4.279a1.008 1.008 0 1 0 1.427-1.427L9.426 8l4.278-4.278a1.008 1.008 0 0 0 0-1.427z" clip-rule="evenodd"/></svg>',
+  'close':'<svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon"><path d="M2.32289 10.1181C1.89237 10.5486 1.89237 11.2466 2.32289 11.6771C2.7534 12.1076 3.45141 12.1076 3.88192 11.6771L7 8.55904L10.1181 11.6771C10.5486 12.1076 11.2466 12.1076 11.6771 11.6771C12.1076 11.2466 12.1076 10.5486 11.6771 10.1181L8.55904 7L11.6771 3.88193C12.1076 3.45141 12.1076 2.7534 11.6771 2.32289C11.2466 1.89237 10.5486 1.89237 10.1181 2.32289L7 5.44096L3.88193 2.32289C3.45141 1.89237 2.7534 1.89237 2.32289 2.32289C1.89237 2.7534 1.89237 3.45141 2.32289 3.88192L5.44096 7L2.32289 10.1181Z" fill="%%COLOR%%"/></svg>',
```
Comment (santhoshbalakrishnan) [inline packages/odyssey/src/scss/abstracts/_functions.scss:78]: Curious, why are viewbox sizes different for different icons? Do we want to standardize on similar viewboxes and modify the content within in relatively?

**Review by santhoshbalakrishnan — APPROVED** [review]
🍰

File: packages/odyssey/src/scss/abstracts/_functions.scss (hunk 4 above)
Comment (edburyenegren-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss:78]: Currently, they're all a little bespoke - @joejezowski-okta designed them for very specific use cases. Agreed that we should work to create some standards here, though. Worth investigating after we revisit Type, @joejezowski-okta @arnoldsandoval-okta?

Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss:78]: Im good with it if Joe is. @joejezowski-okta let me know and I can log an issue for this!

**Comment by arnoldsandoval-okta** [comment]
We were holding off on merging this until #731 was approved. Spoke to @jmelberg-okta and he agreed it was fine to merge this in despite that PR not having final approval as he has signed off for the weekend.

**Pull request merged**